libs/platform/src/lib/components/link/link.component.ts

```diff
@@ -0,0 +1,139 @@
+import {
+    AfterViewInit,
+    Component,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    EventEmitter,
+    ElementRef,
+    Input,
+    OnInit,
+    Output,
+    ViewChild
+} from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+
+export type LinkType = 'standard' | 'emphasized';
+export type NavigationTarget = '_blank' | '_self' | '_parent' | '_top' | 'framename';
+const VALID_INPUT_TYPES = ['standard', 'emphasized'];
+const timeout: number = 1000;
+/** This Timeout should be reviewed */
+
+@Component({
+    selector: 'fdp-link',
+    templateUrl: './link.component.html',
+    styleUrls: ['./link.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LinkComponent implements OnInit, AfterViewInit {
+    private _disabled: boolean = false;
```

Comment:
move private properties at the bottom ( after public ones)

Reply:
moved private properties after public ones

Reply:
Deepak , I mean more under, You inputs , outputs are private properties too.